io.github.sigmasd.share.yml

```diff
@@ -0,0 +1,61 @@
+app-id: io.github.sigmasd.share
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: share
+
+finish-args: 
+  - --share=network
```

Comment:
Why do you need network access?

Reply:
The app :
- spawns a local server # needs network access for this
- accepts file drop
- when the user scans the qr code, he gets the server address

Reply:
Alright